src/frontend/app/features/applications/cli-info-application/cli-info-application.component.ts

```diff
@@ -62,7 +59,7 @@ export class CliInfoApplicationComponent implements OnInit {
     this.route$ = this.store.select(getPreviousRoutingState).pipe(
       map(route => {
         return {
-          url: route && route.state ? route.state.url : defaultBackLink,
```

Comment:
With the close button removed, you can remove the back() method, setupRouteObservable, defaultBackLink etc - these were only there to find the back URL - we are always going back to the summary tab now, so not needed.